projects/distributed-tracing/src/shared/graphql/request/builders/specification/specification-builder.ts

```diff
@@ -48,4 +48,22 @@ export class SpecificationBuilder {
       })
     };
   }
+
+  public fieldSpecificationForKey(field: string): Specification {
+    return {
+      resultAlias: () => field,
+      name: field,
+      asGraphQlSelections: () => ({
+        path: field
+      }),
+      extractFromServerData: serverData => {
+        const serverValue = serverData[field];
+
+        return serverValue === 'null' ? undefined : serverValue;
+      },
+      asGraphQlOrderByFragment: () => ({
```

Comment:
We may want this to throw, since order by is not generally supported for fields.

Reply:
okay let me address this separately